hack/check-docker.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
```

Comment:
I agree with this. We do not need to check whether docker is installed. How about only deleting :
```
@echo "Begin to check dockerd whether is startup"	
./hack/check-docker.sh
```
in the file above.

Reply:
check.sh is unnecessary as well. golangci-lint is the same of it.